resources/etspy-gpu.yml

```diff
@@ -0,0 +1,19 @@
+# environment file for the standard (non-GPU) runtime environment
+name: etspy
+channels:
+  - conda-forge
+  - nvidia
+  - defaults
+dependencies:
+# ensure that cuda version of astra gets installed
+  - astra-toolbox=*=*cuda*
+  - cupy
+  - ffmpeg
+  - hyperspy-base
+  - hyperspy-gui-ipywidgets
+  - ipykernel
+  - pystackreg
+  - pip
+  - numba
+  - numpy<2.0
```

Comment:
Is it necessary to specify `numpy<2.0`, the `pyproject.toml` doesn't specify it https://github.com/usnistgov/etspy/blob/2496f3b39a185a43a0ad21120f059ac0c36fd206/pyproject.toml#L19-L26

Reply:
Thanks, @ericpre. That was an oversight from some testing I was doing with the older version of astra. I'll remove it in the next version